src/v/cluster/controller_backend.cc

```diff
@@ -426,11 +426,13 @@ ss::future<> controller_backend::reconcile_ntp(deltas_t& deltas) {
                         continue;
                     }
                 }
-                vlog(
-                  clusterlog.info,
-                  "partition operation {} result: {}",
-                  *it,
-                  ec.message());
+                if (!it->is_non_replicable()) {
```

Comment:
I could use input here. Maybe theres a cleaner way to not print this log. Also the log below on 439 as well